roles/node/tasks/install-vm.yml

```diff
@@ -26,6 +26,15 @@
     checksum: "sha256:{{ vm_info.download_url }}/v{{ vm_version }}/{{ avalanchego_vm_checksum_filename }}"
     owner: "{{ avalanchego_user }}"
     group: "{{ avalanchego_group }}"
+  when: vm.info is url
```

Comment:
Not sure I get this condition.

Reply:
Messed up my stash sorry was supposed to be `vm.info.download_url is url`

Reply:
We can also use a plain `is defined` for both `vm_info.path` and `vm_info.download_url` in conditions if you'd rather

Reply:
`vm_info`* indeed

Reply:
> We can also use a plain `is defined` for both `vm_info.path` and `vm_info.download_url` in conditions if you'd rather

I switched to this and added a check that both variables are not defined at the same time in the latest push force